### ISSUE TITLE
CSV Import should map to vocabulary item members

### ISSUE THREAD

**Issue by alexdryden** [UIUCLibrary/Digital-Exhibits-Issues-Features#9]
If controlled vocabulary is an item set of items, then when selecting a controlled vocab as a data type for a column in CSV Import those values should be internal omeka links.

**Comment by graykr** [comment]
Yes, I'd like to see this work like the "Omeka resource" data type option where I can specify the title as the field to match on, except that it would only be looking at the items in that custom vocabulary's associated item set. (Since this would be safer than matching on titles for any omeka resource regardless of whether they are in the associated custom vocab or not.)

Right now if I choose a custom vocab that is an item set and try to import, nothing gets imported if I have the titles in the csv.
